Brokerages/Zerodha/ZerodhaBrokerage.cs

```diff
@@ -1060,23 +1061,18 @@ private void OnMessageImpl(MessageData e)
                 }
                 else if (e.MessageType == WebSocketMessageType.Text)
                 {
-                    string message = Encoding.UTF8.GetString(e.Data.Take(e.Count).ToArray());
+                    string textMessage = Encoding.UTF8.GetString(e.Data.Take(e.Count).ToArray());
```

Comment:
`e` should be of type `TextMessage` if `e.MessageType == WebSocketMessageType.Text` right? think it probably throwing an exception at ` var e = (WebSocketClientWrapper.BinaryMessage)message;`, will need a little refactoring I think 👍